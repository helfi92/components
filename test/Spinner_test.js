import React from 'react';
import Spinner from '../src/components/Spinner';

describe('<Spinner />', () => {
  it('should render correctly', () => {
    const spinner = mount(
      <Spinner loading />
    );

    expect(spinner).toMatchSnapshot();
  });
});
