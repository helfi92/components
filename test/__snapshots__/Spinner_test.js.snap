// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Spinner /> should render correctly 1`] = `
<WithStyles(Spinner)
  loading={true}
>
  <Spinner
    className={null}
    classes={
      Object {
        "center": "Spinner-center-1",
      }
    }
    color="primary"
    loading={true}
  >
    <div
      className="Spinner-center-1"
    >
      <WithStyles(CircularProgress)
        className={null}
        color="primary"
      >
        <CircularProgress
          className={null}
          classes={
            Object {
              "circle": "MuiCircularProgress-circle-8",
              "circleDisableShrink": "MuiCircularProgress-circleDisableShrink-11",
              "circleIndeterminate": "MuiCircularProgress-circleIndeterminate-10",
              "circleStatic": "MuiCircularProgress-circleStatic-9",
              "colorPrimary": "MuiCircularProgress-colorPrimary-5",
              "colorSecondary": "MuiCircularProgress-colorSecondary-6",
              "indeterminate": "MuiCircularProgress-indeterminate-4",
              "root": "MuiCircularProgress-root-2",
              "static": "MuiCircularProgress-static-3",
              "svg": "MuiCircularProgress-svg-7",
            }
          }
          color="primary"
          disableShrink={false}
          size={40}
          thickness={3.6}
          value={0}
          variant="indeterminate"
        >
          <div
            className="MuiCircularProgress-root-2 MuiCircularProgress-colorPrimary-5 MuiCircularProgress-indeterminate-4"
            role="progressbar"
            style={
              Object {
                "height": 40,
                "width": 40,
              }
            }
          >
            <svg
              className="MuiCircularProgress-svg-7"
              viewBox="22 22 44 44"
            >
              <circle
                className="MuiCircularProgress-circle-8 MuiCircularProgress-circleIndeterminate-10"
                cx={44}
                cy={44}
                fill="none"
                r={20.2}
                strokeWidth={3.6}
                style={Object {}}
              />
            </svg>
          </div>
        </CircularProgress>
      </WithStyles(CircularProgress)>
    </div>
  </Spinner>
</WithStyles(Spinner)>
`;
